tests/contract-playground/src/parent_chain/ParentChainContract.sol

```diff
@@ -13,7 +13,9 @@ contract ParentChainContract is Ownable, ReentrancyGuard {
     }
 
     function increment(uint256 newNumber) public {
+        // @nyth:blame(constants-instead-of-literals)
         if (number < 5 && number > 1) {
+            // @nyth:blame(constants-instead-of-literals)
```

Comment:
another example

Reply:
What happens when there are multiple instances on the same line (like this one 👀 )?

Reply:
great point ! Should we  make it something like

`@nyth:blame(constants-instead-of-literals:2)`

???

Reply:
( to verify that it was blamed twice )